src/transformers/pipelines/depth_estimation.py

```diff
@@ -96,18 +100,33 @@ def preprocess(self, image, timeout=None):
         return model_inputs
 
     def _forward(self, model_inputs):
-        model_outputs = self.model(**model_inputs)
-        return model_outputs
+        if self.model.config.architectures == ["ZoeDepthForDepthEstimation"]:
+            # That added flipped inference is the default behaviour in the original repo
+            # https://github.com/isl-org/ZoeDepth/blob/edb6daf45458569e24f50250ef1ed08c015f17a7/zoedepth/models/depth_model.py#L99
+            return self.model(
+                pixel_values=torch.cat(
+                    [model_inputs["pixel_values"], torch.flip(model_inputs["pixel_values"], dims=[3])]
+                )
+            )
+
```

Comment:
We shouldn't have model-specific logic in the pipelines. The pipelines are for simple, one-line inference so people can get quickly get predictions. It's OK if they don't perfectly replicate results 

```suggestion
```